lib/reek/constants.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Reek
+  #
+  # This module holds some constants that are used across Reek.
```

Comment:
If we move these up into `Reek` we don't have to namespace them in the rest of the code. WDYT?

Reply:
Yes, great idea, will change that.